tests/phpunit/elementor/includes/base/test-element-base.php

```diff
@@ -178,20 +178,4 @@ public function test_add_invisible_class_attribute_when_animation_on() {
 
 		$this->assertTrue( in_array( 'elementor-invisible', $element->get_render_attributes( '_wrapper', 'class' ) ) );
 	}
-
-	public function test_do_not_add_invisible_class_attribute_when_animation_on_and_render_mode_is_static() {
```

Comment:
Where it moved?

Reply:
Removed for now, cannot test it because for now there is no render_mode class that is static.